sdk/keyvault/Azure.Security.KeyVault.Administration/src/BackupResult.cs

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Azure.Core;
+
+namespace Azure.Security.KeyVault.Administration
+{
+    /// <summary>
+    /// The result of a backup operation.
+    /// </summary>
+    public class BackupResult
+    {
+        internal BackupResult(Uri backupFolderUri, DateTimeOffset startTime, DateTimeOffset endTime)
+        {
+            Argument.AssertNotNull(backupFolderUri, nameof(backupFolderUri));
+
+            this.backupFolderUri = backupFolderUri;
+            StartTime = startTime;
+            EndTime = endTime;
+        }
+        /// <summary>
+        /// The location of the full backup.
```

Comment:
"Gets the ..." (same as below). At least, that's how .NET BCL documentation reads.